packages/eslint-plugin-expo/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "eslint-plugin-expo",
+  "version": "0.0.1",
+  "description": "Eslint rules for Expo apps",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "Expo",
+  "main": "./build/index.js",
+  "files": [
+    "README.md",
+    "build"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/expo/expo.git",
+    "directory": "packages/eslint-plugin-expo"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "test": "jest",
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@typescript-eslint/types": "^7.2.0",
+    "@typescript-eslint/utils": "^7.2.0",
+    "eslint": "^8.57.0"
+  },
+  "devDependencies": {
+    "@types/eslint": "^8.56.5",
+    "@types/jest": "^29.5.12",
+    "@typescript-eslint/rule-tester": "^7.2.0",
+    "eslint": "^8.57.0",
+    "eslint-config-universe": "^12.0.0",
+    "eslint-plugin-eslint-plugin": "^5.0.0",
+    "eslint-plugin-node": "^11.1.0",
+    "jest": "^29.7.0",
+    "prettier": "^3.2.5",
```

```suggestion
    "prettier": "^3.2.5",
```

Prettier also should be put in peer dependencies, and base on the implementation, wo might want ot mark it as optional one too.

---

Prettier was only used to lint _this package_ so I can remove it all together as it's already a dependency of the root package 👍